seq2seq/third_party/models/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 The HuggingFace Team. All rights reserved.
```

Comment:
Can you reply to this comment with the link where you copied this file from? Or did you write it? If you wrote it, remove the HF copyright notice.

Reply:
thanks George, removed.